packages/tasit-action/src/NFT.test.js

```diff
@@ -152,7 +152,9 @@ describe("TasitAction.NFT", () => {
       await confirmBalances([ana.address], [1]);
     });
 
-    it("should transfer an owned token", async () => {
+    // Non-determinisct
```

Comment:
```suggestion
    // Non-deterministic
```

Reply:
Sorry

Reply:
No worries! Haha it's a very unimportant suggested change since it will likely go away soon

Reply:
Done